classes/event/DoubleReputation.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+require_once __DIR__ . '/Event.php';
+
+class DoubleReputation extends Event
```

Comment:
```suggestion
class DoubleReputationEvent extends Event
```
Following convention here